shopelectro/management/commands/price.py

```diff
@@ -1,15 +1,48 @@
-from collections import defaultdict
 import os
+from collections import defaultdict
 
 from django.conf import settings
 from django.core.management.base import BaseCommand
 from django.db.models import Q, QuerySet
 from django.template.loader import render_to_string
 from django.urls import reverse
 
-from shopelectro.models import Product, Category, Tag
+from shopelectro import models
+
+
+# --- files processing ---
+class File:
+    def __init__(self, name: str, dir: str):
+        self.name = name
+        self.dir = dir
+
+    def create(self, context: dict):
+        path = os.path.join(self.dir, self.name)
+        with open(path, 'w', encoding='utf-8') as file:
+            file.write(render_to_string('prices/price.yml', context).strip())
+        return f'{self.name} generated...'
+
 
+class Files:
+    # utm_price_map will be moved to Enum
+    def __init__(self, utm_price_map: dict, base_dir: str):
+        self.utm_price_map = utm_price_map
+        self.base_dir = base_dir
 
+    def create(self):
+        for target, file_name in self.utm_price_map.items():
+            File(file_name, self.base_dir).create(Tree(target).context())
+
+
+class Price:
```

Comment:
What are these classes? How will they act? It would be nice to provide a comment for it to clarify what is going on.

Reply:
@artemiy312 , i'll just rm them.
I already provided `@todo` to continue refactoring